Library/Formula/mpd.rb

```diff
@@ -116,4 +119,11 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
+    pid = fork { exec "#{bin}/mpd --stdout --no-daemon --no-config" }
+    sleep 2
+    Process.kill("SIGINT", pid)
+    Process.wait(pid)
+  end
```

Comment:
Yeah, kinda.

I tried testing it with `curl` but inside Homebrew it vomits permission denied messages at me. If I run the `curl` check outside of Homebrew, it works great. Inside the `test do` block? Failure, so much failure.

And yup, my bad on forgetting the `ensure` :smiley_cat:.